src/xunit.performance.api/CSVMetricReader.cs

```diff
@@ -12,12 +12,11 @@
 
 namespace Microsoft.Xunit.Performance.Api
 {
-    internal class CSVMetricReader
+    internal sealed class CSVMetricReader
     {
-        private Dictionary<string, List<Dictionary<string, double>>> _values = new Dictionary<string, List<Dictionary<string, double>>>();
-
         public CSVMetricReader(string csvPath)
         {
+            _values = new Dictionary<string, List<Dictionary<string, double>>>();
```

Comment:
Preference and consistency with anything else inside this project. I can undo it :)

---
In reply to: [100209662](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100209662) [](ancestors = 100209662)